packages/web-components/src/components/ic-search-bar/ic-search-bar.css

```diff
@@ -112,12 +112,12 @@ ic-menu {
 .search-results-status {
   border: 0;
   clip: rect(0, 0, 0, 0, 0);
-  height: 0.063rem;
-  margin-bottom: -0.063rem;
-  margin-right: -0.063rem;
+  height: var(--ic-space-1px);
+  margin-bottom: -var(--ic-space-1px);
```

Comment:
```suggestion
  margin-bottom: calc(-1 * var(--ic-space-1px));
```

Reply:
Apologies this was missed. This is now resolved